### PR TITLE
content_tunic_campaign_url tags

### PR DESCRIPTION
**Pull request by kand**
Adds `content_tunic_campaign_url` tag for rendering out env-specific tunic campaign urls.

@benghaziboy @MichaelButkovic @mparent61 @daytonn

### CODE REVIEW

**Comment by benghaziboy** [comment]
LGTM

**Comment by MichaelButkovic** [comment]
👍

File: tests/content/test_content_templatetags.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+TEST_TUNIC_BACKEND_ROOT = "//my.tunic.url"
+TEST_TUNIC_API_PATH = "/api/path/"
+
+
+class ContentTemplateTagsTestCase(BaseIndexableTestCase):
+
+    def setUp(self):
+        super(ContentTemplateTagsTestCase, self).setUp()
```
Comment (mparent61) [inline tests/content/test_content_templatetags.py]: Nit: could just delete this since no custom logic added

**Comment by mparent61** [comment]
:+1:

**Pull request merged**